config/filter.d/asterisk.conf

```diff
@@ -20,7 +20,7 @@ log_prefix= (?:NOTICE|SECURITY|WARNING)%(__pid_re)s:?(?:\[C-[\da-f]*\])? [^:]+:\
 
 prefregex = ^%(__prefix_line)s%(log_prefix)s <F-CONTENT>.+</F-CONTENT>$
 
-failregex = ^Registration from '[^']*' failed for '<HOST>(:\d+)?' - (Wrong password|Username/auth name mismatch|No matching peer found|Not a local domain|Device does not match ACL|Peer is not supposed to register|ACL error \(permit/deny\)|Not a local domain)$
+failregex = ^Registration from '[^']*' failed for '<HOST>(:\d+)?' - (Wrong password|Username\/auth name mismatch|No matching peer found|Not a local domain|Device does not match ACL|Peer is not supposed to register|ACL error \(permit\/deny\)|Not a local domain)$
```

Comment:
> I tested and is necessary to process the log.

Can you give more precise statement what you mean, please...
Example log line? resp. log-excerpt? What exactly you've tested?

As already said, it does currently nothing (and for python it is no matter)...
POC:
```bash
$ python -c 'import re; r=r"^(.*)/(.*)$"; print("%s -> %r" % (r, re.search(r, "ABC/123").groups()))'
^(.*)/(.*)$ -> ('ABC', '123')
$ python -c 'import re; r=r"^(.*)\/(.*)$"; print("%s -> %r" % (r, re.search(r, "ABC/123").groups()))'
^(.*)\/(.*)$ -> ('ABC', '123')
```